crates/router/src/cache.rs

```diff
@@ -30,6 +33,10 @@ pub static CONFIG_CACHE: Lazy<Cache> = Lazy::new(|| Cache::new(CACHE_TTL, CACHE_
 pub static ACCOUNTS_CACHE: Lazy<Cache> =
     Lazy::new(|| Cache::new(CACHE_TTL, CACHE_TTI, Some(MAX_CAPACITY)));
 
+/// Using same cache config as above cache.
+pub static KEY_STORE_CACHE: Lazy<Cache> =
```

Comment:
Don't need a seperate static here as well since this and `ACCOUNTS_CACHE`  has same configuration we can use `ACCOUNTS_CACHE`